src/serialization/compact/SchemaService.ts

```diff
@@ -111,11 +111,6 @@ export class SchemaService {
             this.schemas.set(schemaId.toString(), schema);
             return;
         }
-        if (!existingSchema.equals(schema)) {
-            throw new IllegalStateError(
-                `Schema with schemaId ${schemaId} already exists. existing schema: ${existingSchema} new schema: ${schema}`
-            );
-        }
```

Comment:
Why did you delete this?

Reply:
I confused with other control inside put method, I supposed that this was checked inside "put" method. I get it back.